examples/custom_widget.rs

```diff
@@ -53,17 +52,17 @@ impl Widget<String> for CustomWidget {
             (80.0, 90.0),
             (base_state.size().width, base_state.size().height),
         );
-        // Create a solid brush
-        let brush = paint_ctx.solid_brush(Color::rgb24(0x00_80_00));
+        // Create a color
+        let brush = Color::rgb8(0x00, 0x80, 0x00);
```

Comment:
Very minor point, but I wonder if we should call this a "brush".

Reply:
you mean the variable name? or the comment? I don't think it's appropriate in the comment, since 'brush' isn't really a well-defined term. The variable name I don't mind.

Reply:
The variable name. It used to be a brush type, now it's a color, which is "IntoBrush" to be sure, but is not actually a brush.